# Retrieve the list of currently stored prior models
#
# For more details, see the documentation of the
# model_learner_prior node of the articulation_models package,
# http://www.ros.org/wiki/articulation_models
#
---
cob_articulation_controller/ModelMsg[] model
